experimenter/experimenter/experiments/constants.py

```diff
@@ -437,9 +443,8 @@ class EmailType(models.TextChoices):
         will be enrolled in one and not the other and \
         you will not be able to adjust the sizing for this rollout."
 
-    ERROR_DESKTOP_ROLLOUT_VERSION = "WARNING: Decreasing the population size while the \
-        rollout is live is not supported for Desktop versions under 114. You will still \
-        be able to increase the population size."
+    ERROR_ROLLOUT_VERSION = "WARNING: Adjusting the population size while the \
+        rollout is live is not supported for {application} versions under {version}."
```

Comment:
Please reformat without `\`. You can continue strings across multiple lines as so:

```python
ERROR_ROLLOUT_VERSION = (
    "WARNING: ..."
    "...."
)
```

Reply:
Ah that specific message was previously using the backslash so I didn't change it, yeah I'll switch it to the parens